Dockerfile

```diff
@@ -0,0 +1,33 @@
+# BUILD IMAGE --------------------------------------------------------
+FROM nimlang/nim:1.2.0-alpine AS nim-build
```

Comment:
The nimlang version should not be needed.

Reply:
Why is locking a specific Nim version a bad idea?